docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mkdocs
+mkdocs-ansible
```

Comment:
Should we include `pip-tools.toml` and the tooling to pip-compile the `requirements.txt` lockfile?

Reply:
I'm fuzzy on what all that means. Maybe it's better if I merge this PR after addressing other comments and then you can come in and ..erm that pip-compile stuff? or can I just copy 'whatever' you have in the new ecosystem docs repo?

Reply:
yeah we can do that. it makes sense as a follow up PR.